src/ode_solve.jl

```diff
@@ -28,7 +29,8 @@ function DiffEqBase.solve(
     adaptive=false,
     abstol = 1f-6,
     verbose = false,
-    maxiters = 100)
+    maxiters = 100,
+    blackbox = false)

```

Comment:
this shouldn't be needed: if you just pass an `opt` that's for BBO it'll use BBO. Nothing about this package should be different except the ability to pass bounds.